src/Profile/DeviceConfig.cpp

```diff
@@ -80,7 +80,16 @@ LoadPath(const ProfileMap &map, DeviceConfig &config, unsigned n)
 {
   char buffer[64];
   MakeDeviceSettingName(buffer, "Port", n, "Path");
-  return map.Get(buffer, config.path);
+  bool retvalue = map.Get(buffer, config.path);
+#ifdef _WIN32
+  // the usual windows port names has no colon at the end
+  if (retvalue && (config.path.back() == TCHAR(':')) &&
+      config.path.StartsWith(_T("COM"))) {
+    // replace colon with 0, so the path name has the correct length
+    config.path[config.path.length() - 1] = 0;
```

Comment:
Maybe not 0.. because that would result in COM0 which could be mistaken as valid.

Reply:
I changed the character to TCHAR('\0') - maybe that's clearer (and better in UNICODE) - but the result is NOT 'COM0'! e.g. the result of this operation on 'COM1:' is 'COM1' - and that's correct!